src/plugins/bookPlayer/plugin.js

```diff
@@ -16,15 +16,31 @@ import '../../elements/emby-button/paper-icon-button-light';
 import html from './template.html';
 import './style.scss';
 
+const THEMES = {
+    'dark': { 'body': { 'color': '#d8dadc', 'background': '#000', 'font-size': 'medium' } },
+    'sepia': { 'body': { 'color': '#d8a262', 'background': '#000', 'font-size': 'medium' } },
+    'light': { 'body': { 'color': '#000', 'background': '#fff', 'font-size': 'medium' } }
+};
+const THEME_ORDER = ['dark', 'sepia', 'light'];
+const FONT_SIZES = ['x-small', 'small', 'medium', 'large', 'x-large'];
+
 export class BookPlayer {
     constructor() {
         this.name = 'Book Player';
         this.type = 'mediaplayer';
         this.id = 'bookplayer';
         this.priority = 1;
-
+        if (userSettings.theme(undefined) === 'dark' || userSettings.theme(undefined) === null) {
```

Comment:
I know this was a carryover from the existing code, but there's no reason to explicitly pass `undefined` to the `theme()` function.

```suggestion
        if (!userSettings.theme() || userSettings.theme() === 'dark') {
```

Reply:
done!